### ISSUE TITLE
Nimble style, to(beTrue()) or == true

### ISSUE THREAD

**Issue by mokagio** [iflix-letsplay/ios-unit-testing-guidelines#6]
When writing Nimble assertion should we leverage the custom operator for equality, or use the more verbose matcher?

```swift
expect(foo).to(equal(3))
expect(foo) == 3

expect(bar).to(beTrue())
expect(bar) == true
```

From my point of view they both keep the code readable, although from two different angles.

`expect(x).to(equal(y))` actually _reads_ well, like a sentence, which is one of our testing guidelines. On the other hand I feel like `expect(x) == y` is easier on the eye, and makes the code more lean.

Let's vote with a 👍 reaction to the one we prefer, so we can pick one and stick with it 😉 .

**Comment by mokagio** [comment]
### Standard matcher

```swift
expect(foo).to(equal(3))

expect(bar).to(beTrue())
```

**Comment by mokagio** [comment]
### Custom operator matcher

```swift
expect(foo) == 3

expect(bar) == true
```

**Comment by mokagio** [comment]
And yes, this is much of a muchness... But better have this conversation sooner rather than later and get it out of the way.

For the record the topic came up in a private chat between @cocotutch and myself, and I decided to act on it.

**Comment by mokagio** [comment]
We decided as a team to

- Adopt the winning option of "use custom operator whenever possible" as the default choice.
- Do not spend time to point out that `.to(equal(3))` should be `== 3` in code review. We have bigger fishes to fry right now.

<img width="888" alt="screen shot 2018-02-07 at 2 12 25 pm" src="https://user-images.githubusercontent.com/1218433/35896562-f279abf0-0c10-11e8-918a-b774adacc724.png">

Closing.
